docs/summary.md

```diff
@@ -0,0 +1,203 @@
+## Apollo Link
+
+Apollo Link is an extensible standard interface for modifying control flow of GraphQL queries and fetching GraphQL results.
+
+## Basic Usage
+
+Links can be used as a stand-alone client or with most major GraphQL clients.
+
+### Stand-alone
```

Comment:
I'd put a code sample above any actual documentation. So maybe above the `standalone` header.